README.rst

```diff
@@ -25,6 +25,10 @@ Install the dependencies::
 
     pip install -r requirements.txt
 
+.. note::
+
+    You might need to run ``brew install postgres@<VERSION>`` (or a similar command for your package manager) to install PostgreSQL® as a ``psycopg2`` build dependency. 
```

Comment:
Thanks for this - documenting a sensible work-around while we figure out what to actually do sounds sensible,

The brew package name is `postgresql@14` at the moment - which is unusual, as one doesn't normally specify the version. I'd probably say
```
... run ``brew install postgresql@14`` to install PostgreSQL 14 (or a similar command ...
```

Also, GitHub won't render a `note` directive properly, so it's better to do "presentational markup" - that is, something more like:

```
    **Note:** You might need ...
```

(that's indented so it becomes a block quote, which makes it stand out as not part of the main text.